task/python-sdk-azure/0.1/python-sdk-azure.yaml

```diff
@@ -0,0 +1,47 @@
+apiVersion: tekton.dev/v1
+kind: Task
+metadata:
+  name: python-azure-sdk
+  labels:
+    app.kubernetes.io/version: "0.1"
+  annotations:
+    tekton.dev/pipelines.minVersion: "0.54.0"
+    tekton.dev/categories: sdk
+    tekton.dev/tags: CLI, azure-sdk, sdk
+    tekton.dev/displayName: "python azure sdk"
+    tekton.dev/platforms: "linux/amd64"
+spec:
+  params:
+    - name: azure-region
+      type: string
+      default: "eastus"
+      description: Azure Region
+  volumes:
+    - name: python-script
+      configMap:
+        name: python-script-configmap
+  steps:
+    - name: run-python-script
+      image: docker.io/library/python:3.9.19-alpine3.20@sha256:45cc18540209d878c2b24080cf8f64fc37603721b67d0ecc508799e2f9a9b21d
```

Comment:
should we make this image as a parameter. I can see that image used here is 3.9.19 which is old as the latest python version is 3.13

Reply:
yes will do it

Reply:
@vinamra28  done can you check